File: youpinhui-cart-web/src/main/webapp/js/service/cartService.js
//Sevice
app.service('cartService',function($http){


	this.findCartList=function(){
		return $http.get("../cart/findCartList.do");
	}
	
	this.addGoodsToCartList=function(itemId,num){
		return $http.get("../cart/addGoodsToCartList.do?itemId="+itemId+"&number="+num)
	}
  
});

File: youpinhui-cart-interface/src/main/java/com/youpinhui/cart/service/CartService.java
package com.youpinhui.cart.service;

import java.util.List;

import com.youpinhui.pojogroup.Cart;

public interface CartService {
	
	/**
	 * add goods to Cart 
	 * @param list
	 * @param itemId
	 * @param number
	 * @return
	 */
	public List addGoodsToCartList(List<Cart> list,Long itemId,Integer number);	
}
